urbackupserver/www2/src/pages/Status.tsx

```diff
@@ -310,3 +340,35 @@ function transformSelectedRows(selectedRows: Set<TableRowId>) {
   const clientIds = Array.from(selectedRows, Number);
   return clientIds;
 }
+
+function formatDatetime(datetime: number) {
+  return new Date(datetime * 1000).toLocaleString()
+}
+
+function filterClientData(dataItems: StatusClientItem[], search: string) {
+  return dataItems.filter((d) => {
+    // Hide items scheduled for delete
+    if (d.delete_pending === "1") {
+      return false
+    }
+
+    if (search.length) {
+      const { id, name, lastbackup, lastbackup_image } = d
+
+      // Search in fields as displayed in the table
+      const searchableFields = ({
+        id,
+        name,
+        lastbackup: formatDatetime(lastbackup),
+        lastbackup_image: formatDatetime(lastbackup_image)
+      })
+
+      const stringified = JSON.stringify(Object.values(searchableFields)).toLowerCase()
```

Comment:
Not much of a problem perhaps but that would include some extra characters like `["`. Maybe we can just something similar to `.join("")` instead of using JSON?

Reply:
Hmm, join might have other problems. Perhaps a simple for loop would be better?

Reply:
Thanks for pointing that out! New commit loops over the values to find matching search term.